src/fluree/db/util/filesystem.cljc

```diff
@@ -78,6 +78,22 @@
                             "code"    (.-code e)
                             "path"    (.-path e)})))))))
 
+(defn delete-file
+  "Delete the file at `path`."
+  [path]
+  #?(:clj
+     (try
+       (io/delete-file (io/file path))
+       :deleted
+       (catch Exception e
```

Comment:
i don't think we should be catching exceptions here so we can allow downstream consumers of this api to handle errors as they arise.